client/js/components/fragments/Map.js

```diff
@@ -42,17 +42,17 @@ class Map extends Component {
         google.maps.MapTypeId.SATELLITE,
         google.maps.MapTypeId.TERRAIN
       ],
-      scaleControl: true,
-      mapTypeControl: true,
+      scaleControl: false,
```

Comment:
Why did you disabled all the controls? They are really useful, especially the possibility to switch map type. Zooming out with touchpad is also not really obvious.